remo/featuredrep/admin.py

```diff
@@ -25,5 +25,13 @@ class FeaturedRepAdmin(ExportMixin, admin.ModelAdmin):
     def featured_rep_users(self, obj):
         return ', '.join([user.get_full_name() for user in obj.users.all()])
 
+    def save_model(self, request, obj, form, change):
+        super(FeaturedRepAdmin, self).save_model(request, obj, form, change)
+        featured = FeaturedRep.objects.get(pk=obj.pk)
+
+        if (form.is_valid() and 'updated_on' in form.changed_data and
```

Comment:
I'm a bit confused by this, and wasn't able to test it as I'm not sure how to log in to the admin with the demo admin user. As you are calling `save` on the parent before this code, won't the code from line 34 in the model have already caused `updated_on` to be set to the value from the form, if one was specified? And if that is true, then when would this statement ever evaluate to `True`. Please forgive me if I'm being dense -  I'm just trying to understand the code.